http/security/helper.go

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/30 by Charlotte Pröller
+
+package security
+
+import (
+	"context"
+	"net/http"
+	"strings"
+)
+
+type Token interface {
+	GetValue() string
+}
+
+type bearerToken struct {
+	value string
+}
+
+type Ctxkey string
+
+const HeaderPrefix = "Bearer "
+
+var TokenKey = Ctxkey("Token")
```

Comment:
`Ctxkey` and `TokenKey` should not be exported. See https://golang.org/pkg/context/#Context

> A key identifies a specific value in a Context. Functions that to store values in Context typically allocate a key in a variable then use that key as the argument to context.WithValue and Context.Value. A key can be any type that supports equality; packages should define keys as an unexported type to avoid collisions.